includes/class-freemius.php

```diff
@@ -5155,19 +5175,43 @@ private function parse_settings( &$plugin_info ) {
                 $this->_plugin :
                 new FS_Plugin();
 
+            $is_premium     = $this->get_bool_option( $plugin_info, 'is_premium', true );
             $premium_suffix = $this->get_option( $plugin_info, 'premium_suffix', '(Premium)' );
 
+            $module_type = $this->get_option( $plugin_info, 'type', $this->_module_type );
+
+            $parallel_activation = $this->get_option( $plugin_info, 'parallel_activation' );
+
+            if (
+                ! $is_premium &&
+                ( WP_FS__MODULE_TYPE_PLUGIN === $module_type ) &&

```

Comment:
Micro performance, given we can assume the `plugin` type is most common, yet parallel activation would be fairly uncommon:

```php
if (
  ! $is_premium &&
  is_array( $parallel_activation ) &&
  ...
)
```